javascript/exercise_form/signUp.js

```diff
@@ -0,0 +1,136 @@
+const email = document.querySelector('.email');
+const username = document.querySelector('.username');
+const password = document.querySelector('.password');
+const confirmPassword = document.querySelector('.confirm-password');
+const signUpForm = document.getElementById('sign-up-form');
+let isErrors = false;
+
+function checkErrors(element) {
+  const error = element.parentElement.querySelector('.message');
+
+  if (isErrors) {
+    element.classList.add('valid');
+    error.style.display = 'block';
+  } else {
+    element.classList.remove('valid');
+    error.style.display = 'none';
+  }
+}
+
+const checkEmail = () => {
```

Comment:
Should be isValidEmail, because the result is boolean => so we should naming base on the result return.